github_copilot_seats/seats.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+from datetime import datetime, timezone, timedelta
+from typing import Any
+import autokitteh
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+from helpers import github_username_to_slack_user_id
+
+
+GITHUB_ORG = os.getenv("GITHUB_ORG")
```

Comment:
How to get the GitHub org name without a manually-specified project var - it's already a connection var:

https://github.com/autokitteh/kittehub/pull/139/files#diff-81e3c3caea8e0d76885489976911b513696a4ee6b1084e874f4f6ae0a4c37216